### PR TITLE
Initial Creation

### PR DESCRIPTION
**Pull request by ghost**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,5 +23,8 @@
   "devDependencies": {
     "dtslint": "github:Microsoft/dtslint#production",
     "types-publisher": "Microsoft/types-publisher#production"
+  },
+  "dependencies": {
+    "openxml": "^1.0.2"
   }
```
Comment (plantain-00) [inline package.json]: You shouldn't change this file, it is used by ts files in `script/`

**Review by plantain-00 — CHANGES_REQUESTED** [review]

**Comment by RyanCavanaugh** [comment]
@PeterJohnHanley Please address comments from the code reviewers.

File: package.json (hunk 1 above)
Comment (PeterJohnHanley) [reply package.json]: Not interested in your silly crap!!!

**Comment by DanielRosenwasser** [comment]
This is a 72K line long contribution; is there any possibility that you can reduce the identical properties to a single set of utility types?

For example, there are so many occurrences of

```ts
{
                localName: string;
                namespace: {
                    getName: any;
                    namespaceName: string;
                    preferredPrefix: any;
                    toString: any;
                };
                namespaceName: string;
                toString: any;
            };
```

and many more of

```ts
{
                    getName: any;
                    namespaceName: string;
                    preferredPrefix: any;
                    toString: any;
                };
```

So you could define

```ts
interface StandardNamespace {
    getName: any;
    namespaceName: string;
    preferredPrefix: any;
    toString: any;
}

interface WhateverThisIs {
    localName: string;
    namespaceName: string;
    toString: any;
    namespace: StandardNamespace;
}
```

File: package.json (hunk 1 above)
Comment (DanielRosenwasser) [reply package.json]: Hard to tell if you're joking around, but please address this feedback. 😃

**Comment by PeterJohnHanley** [comment]
Not joking
If an initial creation is so hard and DELAYED!!, it can only get worse!!
Started rewriting small app in C# and  .NET MVC
Don’t worry, you won’t hear from me again!
Peter


Sent from Mail for Windows 10

**Pull request closed without merging**